internal/cmd/freeze-api/convert.go

```diff
@@ -149,3 +160,7 @@ func typesEquivalent(a, b *ast.TypeSpec) bool {
 
 	return t1 == t2
 }
+
+func packagePath(path string) string {
+	return filepath.Join(build.Default.GOPATH, "src", path)
+}
```

Comment:
This is already defined in the `astutil` package.

Reply:
True. I can export it there and remove this definition.